spec/rubocop/cop/style/empty_literal_spec.rb

```diff
@@ -31,19 +31,19 @@
     it 'does not registers an offence Array.new with block' do
       source = 'test = Array.new { 1 }'
       inspect_source(cop, source)
-      expect(cop.offenses.size).to eq(0)
+      expect(cop.offenses).to be_empty
     end
 
     it 'registers an offence when Array.new in block' do
       source = 'puts { Array.new }'
-      inspect_source(cop, source)
-      expect(cop.offenses.size).to eq(1)
+      new_source = autocorrect_source(cop, source)
```

Comment:
This test is just supposed to check if there's offense in the code or not, it's not supposed to check if the auto-correct works.